custom.conf.template

```diff
@@ -0,0 +1,8 @@
+[Labels]
+required-labels-any: # Either provide a list of comma-separated labels, or leave blank if not using
```

Comment:
You could just add this comment at the top of the `[Labels]` section to prevent repeating this for each option.